src/backend/core/api/viewsets.py

```diff
@@ -188,9 +188,16 @@ class UserViewSet(
     permission_classes = [permissions.IsSelf]
     queryset = models.User.objects.all().order_by("-created_at")
     serializer_class = serializers.UserSerializer
+    me_serializer_class = serializers.UserMeSerializer
```

Comment:
Reminds me that we have this SerializerPerAction mixin: https://github.com/numerique-gouv/people/pull/509/files#diff-cc27a5de6f1d8131250e4be30d0fd4c2f1a816bdc35951a9c86e6f13cd795019R82